.circleci/create_circleci_config.py

```diff
@@ -51,6 +51,7 @@ class CircleCIJob:
     resource_class: Optional[str] = "xlarge"
     tests_to_run: Optional[List[str]] = None
     working_directory: str = "~/transformers"
+    timeout :int = None
```

Comment:
could we add a comment to mention what it is for?
(i.e. not to confuse the readers with `pytest timeout` and this timeout)